core/common/src/TextureProps.ts

```diff
@@ -25,13 +30,11 @@ export interface TextureProps extends DefinitionElementProps {
    * @see [[Base64EncodedString.toUint8Array]] to decode the bytes.
    */
   data: Base64EncodedString;
-  /** The width of the image. */
+  /** The width of the image. Do not rely on the value of this property - it may not match the actual image width. */

```

Comment:
If you can't rely on it, what value does it have? Why don't they match?

Reply:
Every .bim file I've checked has zero for the width and height columns. It would appear native code does not set them properly. We could of course fix native code, but that would not help with existing iModels.

Reply:
If the value is non-zero, is it correct? If so, say "if non-zero,..."

Reply:
We have to make them optional then too. Why don't we just delete them?
Also, why don't we just delete the `flags` property that is always zero and used for nothing and has a stupid bug in native code:
```
    if (props.isMember(json_height()))
        m_flags = (Flags) props[json_flags()].asUInt();
```

Reply:
how bout we mark them @deprecated?